style.css

```diff
@@ -10,6 +10,146 @@
   font-family: 'inter', sans-serif;
 }
 
+.modal-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.modal-social li {
+  box-sizing: border-box;
+  align-items: center;
+  padding: 10px 15px;
+  border: 1px solid #8993a4;
+  margin: 0 5px;
+}
+
+.live-button {
+  background-color: #ff6b00;
+  border: none;
+  color: #fff;
+  font-family: "Inter", sans-serif;
+  font-size: 13px;
+  display: flex;
+  text-align: center;
+  padding: 6.5px 13px;
+  gap: 25px;
+}
+
+.source-button {
+  background-color: #ff6b00;
+  border: none;
+  color: #fff;
+  font-family: "Inter", sans-serif;
+  font-size: 13px;
+  display: flex;
+  text-align: center;
+  padding: 8px 13px;
+  gap: 15px;
+}
+
+.modal {
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  z-index: 10;
+  background-color: #fff;
+  width: 90%;
+  height: 95%;
+  border: 1px solid #dfe1e6;
+  animation: 1s drop;
+  display: none;
+}
+
+@keyframes drop {
+  0% {
+    top: -100px;
+  }
+
+  100% {
+    top: 50%;
+  }
+}
+
+.title {
+  font-family: "Roboto", sans-serif;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 44px;
+  color: #172b4d;
+  margin: 24px;
+  padding-bottom: 20px;
+}
+
+.modal-headd {
+  display: none;
+}
+
+.close-button {
+  cursor: pointer;
+  outline: none;
+  margin-right: 24px;
+  background-color: #fff;
+  border: none;
+  font-size: 30px;
+  transition: 200ms ease-in-out;
+  margin-bottom: 44px;
+}
+
+.modal-social {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-size: 13px;
+  padding-left: 24px;
+  font-weight: 600;
+  margin-top: 15px;
+}
+
+.modal-pic {
+  display: flex;
+  flex-direction: column;
+  background-image: url(./images/SnapshootPortfolio.png);
+  background-repeat: no-repeat;
+  background-size: 90% 100%;
+  margin-top: 10px;
+  margin-left: 24px;
+}
+
+.modal-body {
+  font-family: "Inter", sans-serif;
+  font-size: 17px;
+  line-height: 22px;
+  height: 5px;
+  color: #344563;
+  margin-top: 150px;
+}
+
+.modal-mobile {
+  padding: 30px 0;
+  padding-right: 30px;
+  left: 0;
+  right: 0;
+}
+
+.modal-desktop {
+  display: none;
+}
+
+.modal-button {
+  display: flex;
+  flex-direction: row;
+  gap: 90px;
+  padding-bottom: 82px;
+  max-width: 350px;
+  height: 580px;
+  margin-left: 24px;
+  align-items: center;
+}
```

Comment:
- [ ] Great job here 💯 However, According to the requirements You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So kindly make the modal-buttons below the paragraph 👍

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/189284334-d7bc2075-28b9-47b1-819e-1abcb7e19f3c.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/189284356-0e5ac33b-1d53-4eb0-9962-3b09c9c668ff.PNG" width="500" /> |